skimage/future/graph/rag.py

```diff
@@ -498,7 +498,7 @@ def show_rag(labels, rag, image, border_color='black', edge_width=1.5,
     >>> lc = graph.show_rag(labels, g, img)
     >>> cbar = plt.colorbar(lc)
     """
-    from matplotlib import colors, colormaps
+    from matplotlib import colors, pyplot
```

Comment:
```suggestion
    from matplotlib import colors
```
`pyplot` is already imported as `plt` in the line below :wink:

Reply:
Ha, yes. Good catch! :sweat_smile: